components/search/__tests__/list-renderer.test.js

```diff
@@ -1220,3 +1222,115 @@ describe("Test the MeasurementSet component", () => {
     expect(status).toHaveTextContent("released");
   });
 });
+
+describe("Test the Software component", () => {
+  it("renders a Software item", () => {
+    const item = {
+      "@id": "/software/bowtie2/",
+      "@type": ["Software", "Item"],
+      name: "bowtie2",
+      title: "Bowtie2",
+      lab: {
+        title: "J. Michael Cherry, Stanford",
+      },
+      status: "released",
+    };
+
+    render(
+      <SessionContext.Provider value={{ profiles }}>
+        <Software item={item} />
+      </SessionContext.Provider>
+    );
+
+    const uniqueId = screen.getByTestId("search-list-item-unique-id");
+    expect(uniqueId).toHaveTextContent(/^Software/);
+    expect(uniqueId).toHaveTextContent(/bowtie2$/);
+
+    const title = screen.getByTestId("search-list-item-title");
+    expect(title).toHaveTextContent(/^Bowtie2$/);
+
+    const meta = screen.getByTestId("search-list-item-meta");
+    expect(meta).toHaveTextContent("J. Michael Cherry, Stanford");
+
+    const status = screen.getByTestId("search-list-item-status");
+    expect(status).toHaveTextContent("released");
+  });
+});
+
+describe("Test the SoftwareVersion component", () => {
+  it("renders a SoftwareVersion item with accessory data", () => {
+    const item = {
+      "@id": "/software-versions/bowtie2-v2.4.4/",
+      "@type": ["SoftwareVersion", "Item"],
+      software: {
+        title: "Bowtie2",
+      },
+      version: "2.4.4",
+      lab: "/labs/j-michael-cherry/",
+      status: "released",
+      uuid: "67380d9f-06da-f9fe-9569-d31ce0607eae",
+    };
+    const accessoryData = {
+      "/labs/j-michael-cherry/": {
+        "@id": "/labs/j-michael-cherry/",
+        "@type": ["Lab", "Item"],
+        title: "J. Michael Cherry, Stanford",
+      },
+    };
+
+    render(
+      <SessionContext.Provider value={{ profiles }}>
+        <SoftwareVersion item={item} accessoryData={accessoryData} />
+      </SessionContext.Provider>
+    );
+
+    const uniqueId = screen.getByTestId("search-list-item-unique-id");
+    expect(uniqueId).toHaveTextContent(/^Software/);
+    expect(uniqueId).toHaveTextContent(/Bowtie2 2.4.4$/);
+
+    const title = screen.getByTestId("search-list-item-title");
+    expect(title).toHaveTextContent(/^Bowtie2 2.4.4$/);
+
+    const meta = screen.getByTestId("search-list-item-meta");
+    expect(meta).toHaveTextContent("J. Michael Cherry, Stanford");
+
+    const status = screen.getByTestId("search-list-item-status");
+    expect(status).toHaveTextContent("released");
+
+    const paths = MeasurementSet.getAccessoryDataPaths([item]);
```

Comment:
Change this to `SoftwareVersion.getAccessoryDataPaths([item])` and you’ll get full Jest coverage.